src/runtime_src/core/common/api/xrt_module.cpp

```diff
@@ -156,6 +157,19 @@ struct patcher
     bd_data_ptr[3] = (bd_data_ptr[3] & 0xFFFF0000) | (base_address >> 32);
   }
 
+  void patch_shim48(uint32_t* bd_data_ptr, uint64_t patch)
+  {
+    constexpr uint64_t DDR_AIE_ADDR_OFFSET = ((uint64_t)(0x80000000));
```

Comment:
Avoid all caps.
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#nl9-use-all_caps-for-macro-names-only